src/editor/Fgd.cpp

```diff
@@ -294,7 +294,7 @@ vector<string> Fgd::groupParts(vector<string>& ungrouped) {
 
 		if (stringGroupStarts(ungrouped[i])) {
 			string groupedPart = ungrouped[i];
-
+			// FIXME : warn 'i' used in prev loop

```

Comment:
It's on purpose, to continue searching for new string groups after the end of this string group.

Reply:
Ok